doc/sphinx/deploying.rst

```diff
@@ -21,7 +21,7 @@ Production settings
 
 As any Django website, Dissemin can be served by various web servers.
 These settings are not specific to dissemin itself so you should refer
-to `the relevant Django documentation <https://docs.djangoproject.com/en/1.11/howto/deployment/>`_.
+to `the relevant Django documentation <https://docs.djangoproject.com/en/2.2/howto/deployment/>`_.
```

Comment:
Not sure about the relevant Django documentation here, with https://github.com/dissemin/dissemin/issues/611 in mind. So far, dissem.in is in 2.0.8 I think, but the issue is not solved yet :/

Same below when linking to Django doc.

Reply:
I will investigate and fix this shortly

Reply:
Fixed, the docs are still current.